db/schema.rb

```diff
@@ -413,7 +413,6 @@
     t.datetime "updated_at"
   end
 
-  add_index "repository_rows", ["name"], name: "index_repository_rows_on_name", using: :btree
```

Comment:
why is this line deleted?

Reply:
Not sure, i didnt do this, i thought a rake db command did it.